src/prefect/schedules/filters.py

```diff
@@ -181,3 +181,40 @@ def is_weekend(dt: datetime) -> bool:
         - bool: True if the date is a weekend; False otherwise
     """
     return pendulum.instance(dt).weekday() > 4
+
+
+def is_month_start_or_specific_day(day_of_week: int) -> Callable[[datetime], bool]:
```

Comment:
Since we support OR operations for filters, why combine these into a single filter? Seems to me it might be more composable/easier to have:

- `is_month_start` - is it the start of a month. We already have `is_month_end`, so this follows the existing pattern
- `is_day_of_week(n)` - is it a specific day of the week.

`is_month_start_or_specific_day` could then be built out of these smaller blocks.

Reply:
Yeah that's fair, I can separate them @jcrist

Reply:
Ok that's done and tests are updated, just waiting on those to run 👍